src/engine/TransitivePathHashMap.cpp

```diff
@@ -3,31 +3,89 @@
 // Author: Florian Kramer (florian.kramer@neptun.uni-freiburg.de)
 //         Johannes Herrmann (johannes.r.herrmann(at)gmail.com)
 
-#include "TransitivePathHashMap.h"
+#include "engine/TransitivePathHashMap.h"
 
 #include <memory>
 
 #include "engine/CallFixedSize.h"
 #include "engine/TransitivePathBase.h"
 
 // _____________________________________________________________________________
-HashMapWrapper TransitivePathHashMap::setupEdgesMap(
-    const IdTable& dynSub, const TransitivePathSide& startSide,
-    const TransitivePathSide& targetSide) const {
-  return CALL_FIXED_SIZE((std::array{dynSub.numColumns()}),
-                         &TransitivePathHashMap::setupEdgesMap, this, dynSub,
-                         startSide, targetSide);
+HashMapWrapper::HashMapWrapper(
+    Map map, const ad_utility::AllocatorWithLimit<Id>& allocator)
+    : graphMap_{allocator},
+      map_{nullptr},
```

Comment:
Can't you directly construct here (i am sure that `unordered_map` has a constructor that takes allocator + initializer_list.)